core/backend/src/core-backend.ts

```diff
@@ -76,6 +74,8 @@ export * from "./workspace/WorkspaceEditor";
 export * from "./SqliteChangesetReader";
 export * from "./ChangesetECAdaptor";
 
+export * from "./internal/cross-package";
```

Comment:
This is what will be removed in 5.0 but is here right now to not break potential usage, correct?

Reply:
Yes. Also because we currently don't yet have a solution for knowing whether to import from lib/cjs or lib/esm. For 5.0 we either need to ditch cjs, or find a way to omit the cross-package.ts export from the published packages.